src/main/java/com/example/java/maven/gameBlackJack/Deck.java

```diff
@@ -0,0 +1,33 @@
+package com.example.java.maven.gameBlackJack;
+
+import java.util.Collections;
+import java.util.Stack;
+
+public class Deck {
+    private final Stack<Card> cards;
+
+    public Deck() {
+        this.cards = new Stack<>();
+        for (Card.Suit suit : Card.Suit.values()) {
+            for (Card.Rank rank : Card.Rank.values()) {
+                Card card = new Card(suit, rank);
+                cards.push(card);
+            }
+        }
+        Collections.shuffle(cards);
+    }
+
+    public Stack<Card> getCards() {
```

Comment:
we were talking about it. If you need return `cards` return copy of it. But IMHO this method is not necessary

Reply:
Done